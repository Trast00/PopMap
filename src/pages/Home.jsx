import React from 'react'
import Header from '../components/Header'

const Home = () => {
  return (
    <div>
      <Header name="Home" canBack={false} />
      Page Home
    </div>
  )
}

export default Home