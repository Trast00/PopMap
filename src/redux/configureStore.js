import { configureStore } from '@reduxjs/toolkit';
import countryReducer from './country/countryReducer';

export default configureStore({
  reducer: {
    countryReducer,
  },
});
