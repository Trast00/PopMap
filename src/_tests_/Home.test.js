import renderer from 'react-test-renderer';
import '@testing-library/jest-dom';
import React from 'react';
import { Provider } from 'react-redux';
import configureStore from '../redux/configureStore';
import Home from '../pages/Home';

describe('test render', () => {
  test('Home should match snapshoot', () => {
    const tree = renderer.create(
      <React.StrictMode>
        <Provider store={configureStore}>
          <Home />
        </Provider>
      </React.StrictMode>,
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
