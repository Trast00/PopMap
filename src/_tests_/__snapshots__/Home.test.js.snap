// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test render Home should match snapshoot 1`] = `
<div
  className="max-width"
>
  <header
    className="flex-center header"
  >
    <div />
    <p>
      Home
    </p>
    <div>
      <svg
        className="icons"
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        strokeWidth="0"
        style={
          Object {
            "color": undefined,
          }
        }
        viewBox="0 0 16 16"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M5 3a3 3 0 0 1 6 0v5a3 3 0 0 1-6 0V3z"
        />
        <path
          d="M3.5 6.5A.5.5 0 0 1 4 7v1a4 4 0 0 0 8 0V7a.5.5 0 0 1 1 0v1a5 5 0 0 1-4.5 4.975V15h3a.5.5 0 0 1 0 1h-7a.5.5 0 0 1 0-1h3v-2.025A5 5 0 0 1 3 8V7a.5.5 0 0 1 .5-.5z"
        />
      </svg>
      <svg
        className="icons"
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        strokeWidth="0"
        style={
          Object {
            "color": undefined,
          }
        }
        viewBox="0 0 512 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M416.3 256c0-21 13.1-38.9 31.7-46.1-4.9-20.5-13-39.7-23.7-57.1-6.4 2.8-13.2 4.3-20.1 4.3-12.6 0-25.2-4.8-34.9-14.4-14.9-14.9-18.2-36.8-10.2-55-17.3-10.7-36.6-18.8-57-23.7C295 82.5 277 95.7 256 95.7S217 82.5 209.9 64c-20.5 4.9-39.7 13-57.1 23.7 8.1 18.1 4.7 40.1-10.2 55-9.6 9.6-22.3 14.4-34.9 14.4-6.9 0-13.7-1.4-20.1-4.3C77 170.3 68.9 189.5 64 210c18.5 7.1 31.7 25 31.7 46.1 0 21-13.1 38.9-31.6 46.1 4.9 20.5 13 39.7 23.7 57.1 6.4-2.8 13.2-4.2 20-4.2 12.6 0 25.2 4.8 34.9 14.4 14.8 14.8 18.2 36.8 10.2 54.9 17.4 10.7 36.7 18.8 57.1 23.7 7.1-18.5 25-31.6 46-31.6s38.9 13.1 46 31.6c20.5-4.9 39.7-13 57.1-23.7-8-18.1-4.6-40 10.2-54.9 9.6-9.6 22.2-14.4 34.9-14.4 6.8 0 13.7 1.4 20 4.2 10.7-17.4 18.8-36.7 23.7-57.1-18.4-7.2-31.6-25.1-31.6-46.2zm-159.4 79.9c-44.3 0-80-35.9-80-80s35.7-80 80-80 80 35.9 80 80-35.7 80-80 80z"
        />
      </svg>
    </div>
  </header>
  <div
    className="max-width flex-center continent"
  >
    <div
      className="flex-center column country detail-country"
    >
      <img
        alt="WorldCountry Images"
        src="world.svg"
      />
      <div
        className="column detail-wrapper"
      >
        <h4>
          World
        </h4>
        <p>
          Citizens: 
           
          0
        </p>
      </div>
    </div>
    <svg
      className="icons filter-next"
      fill="currentColor"
      height="1em"
      onClick={[Function]}
      stroke="currentColor"
      strokeWidth="0"
      style={
        Object {
          "color": undefined,
        }
      }
      viewBox="0 0 512 512"
      width="1em"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M256 8c137 0 248 111 248 248S393 504 256 504 8 393 8 256 119 8 256 8zm-28.9 143.6l75.5 72.4H120c-13.3 0-24 10.7-24 24v16c0 13.3 10.7 24 24 24h182.6l-75.5 72.4c-9.7 9.3-9.9 24.8-.4 34.3l11 10.9c9.4 9.4 24.6 9.4 33.9 0L404.3 273c9.4-9.4 9.4-24.6 0-33.9L271.6 106.3c-9.4-9.4-24.6-9.4-33.9 0l-11 10.9c-9.5 9.6-9.3 25.1.4 34.4z"
      />
    </svg>
  </div>
  <div
    className="max-width"
  >
    <ul
      className="max-width no-styling row list-country"
    >
      <p
        className="max-width flex-center"
      >
        Empty List
      </p>
    </ul>
  </div>
</div>
`;
